mw-webapp/src/router/ProtectedRouter.tsx

```diff
@@ -0,0 +1,20 @@
+import {Navigate, Outlet, useLocation} from "react-router-dom";
+
+/**
+ * Protected Router
+ */
+export const ProtectedRouter = () => {
+  const location = useLocation();
+  const history = localStorage.getItem("auth") ?? "";
+  const is = JSON.parse(history);
```

Comment:
Rename variables please (authStatus, isAuth and etc)

Reply:
done

Reply:
done